easybuild/easyconfigs/e/EMBOSS/EMBOSS-6.6.0-foss-2021a.eb

```
@@ -0,0 +1,58 @@
+# authors: Kenneth Hoste (Ghent University)
+#          George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+#          Fotis Georgatos <fotis@cern.ch>
+#
+# This work implements a part of the HPCBIOS project and is a component
+#  of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# foss-2016b modified by:
+# Adam Huffman
+# The Francis Crick Institute
+# foss-2021a modified by:
+# Sebastien Moretti
+# SIB Swiss Institute of Bioinformatics
+easyblock = 'ConfigureMake'
+
+name = 'EMBOSS'
+version = '6.6.0'
+
+homepage = 'http://emboss.sourceforge.net/'
+description = """EMBOSS is 'The European Molecular Biology Open Software Suite'
+. EMBOSS is a free Open Source software analysis package specially developed
+ for the needs of the molecular biology (e.g. EMBnet) user community."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = [
+    'ftp://emboss.open-bio.org/pub/EMBOSS/',
+    'ftp://emboss.open-bio.org/pub/EMBOSS/old/%(version_major_minor)s.0',
+]
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s_disable-embossupdate.patch']
+checksums = [
+    '7184a763d39ad96bb598bfd531628a34aa53e474db9e7cac4416c2a40ab10c6e',  # EMBOSS-6.6.0.tar.gz
+    '7e0a7deffd76f60093be9c5253605f2d6d2e3b0c2d3c9365035cc6bda43eb46c',  # EMBOSS-6.6.0_disable-embossupdate.patch
+]
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('X11', '20210518'),
+    ('libharu', '2.3.0'),
+    ('Java', '11.0.2', '', True),
```

@smoretti This should be the `Java/11` wrapper, rather than a specific version:

```suggestion
    ('Java', '11', '', True),

---

Done